modules/nf-core/iqtree/main.nf

```diff
@@ -52,7 +52,7 @@ process IQTREE {
 
     script:
     def args                        = task.ext.args           ?: ''
-    def alignment_arg               = alignment               ? "-s $alignment"                 : ''
+    def alignment_arg               = alignment               ? "-s ${alignment.join(',')}"     : ''
```

Comment:
Nice to have (but not a blocker): a test case with multiple input alignments :)

Reply:
Yes, I looked into that but could not find test data that would work already in `test-datasets`. We would need two alignments with different names but the same headers.